aws/cluster/templates/cluster-spec.yaml

```diff
@@ -23,22 +23,23 @@ ${cloud-labels}
 ${etcd-clusters}
   keyStore: s3://${kops-state-bucket}/${cluster-name}/pki
   kubeAPIServer:
-    address: 127.0.0.1
-    admissionControl:
```

Comment:
Ahaaa enableAdmissionPlugins probably does not have order but admissionControl did. Do we have an option of setting up the order?

Reply:
Than it should be OK